### PR TITLE
WinVersion class follow-up: remove ease of access is supported flag

### PR DESCRIPTION
**Pull request by josephsl**
### Link to issue number:
Follow-up to PR #11909

### Summary of the issue:
NVDA requires Windows 7/Server 2008 R2 SP1 or later, therefore Ease of Access is available.

### Description of how this pull request fixes the issue:
Removes easeOfAccess.isSupported flag.

### Testing strategy:
Code review (Grep used), manual testing to make sure easeOfAccess.isSupported raises AttributeError.

### Known issues with pull request:
None

### Change log entry:
Changes for developers:

easeOfAccess.isSupported has been removed.

### Code Review Checklist:

This checklist is a reminder of things commonly forgotten in a new PR.
Authors, please do a self-review and confirm you have considered the following items.
Mark items you have considered by checking them.
You can do this when editing the Pull request description with an x: `[ ]` becomes `[x]`.
You can also check the checkboxes after the PR is created.

- [x] Pull Request description is up to date.
- [x] Unit tests.
- [x] System (end to end) tests.
- [x] Manual tests.
- [x] User Documentation.
- [x] Change log entry.
- [x] Context sensitive help for GUI changes.

Thanks.

### CODE REVIEW

File: source/config/__init__.py [hunk 1]
```diff
@@ -239,13 +241,9 @@ def getStartOnLogonScreen():
...
 def _setStartOnLogonScreen(enable):
-	if easeOfAccess.isSupported:
-		# The installer will have migrated service config to EoA if appropriate,
-		# so we only need to deal with EoA here.
-		easeOfAccess.setAutoStart(winreg.HKEY_LOCAL_MACHINE, enable)
-	else:
-		k = winreg.OpenKey(winreg.HKEY_LOCAL_MACHINE, NVDA_REGKEY, 0, winreg.KEY_WRITE)
-		winreg.SetValueEx(k, u"startOnLogonScreen", None, winreg.REG_DWORD, int(enable))
+	# The installer will have migrated service config to EoA if appropriate,
+	# so we only need to deal with EoA here.
```
Comment (seanbudd) [inline source/config/__init__.py:245]:
```suggestion
```

I don't think these comments make sense now that the else condition is removed

**Review by seanbudd — CHANGES_REQUESTED** [review]
Thanks @josephsl. Looks good to me, can also confirm there are no other usages after merging in master (e019a243271a45816223c5f4fa661cda4b75d9fe).

**Comment by josephsl** [comment]
Hi, I see. I’ll address this as early as upcoming weekend. Thanks.

**Comment by josephsl** [comment]
Hi,

Review comments addressed.

Thanks.

**Review by seanbudd — APPROVED** [review]
Thanks @josephsl , LGTM

**Pull request merged**